ci/tests/test_00_unit/test_05_index_ci/test_00_schema_validation/test_06_git_url_validation/__init__.py

```diff
@@ -0,0 +1,40 @@
+from ci.tests.test_00_unit.test_05_index_ci.indexcibase import IndexCIBase, \
+    DUMMY_INDEX_FILE
+from ci.container_index.lib.constants import *
+import ci.container_index.lib.checks.schema_validation as schema_validation
+
+
+class GitUrlValidationTests(IndexCIBase):
+
+    def test_00_setup_test(self):
+        self._setup_test()
+
+    def test_01_validation_succeeds_valid_job_id(self):
+        self.assertTrue(
+            schema_validation.GitURLValidator(
+                {
+                    FieldKeys.GIT_URL: "https://github.com/test/test"
+                },
+                DUMMY_INDEX_FILE
+            ).validate().success
+        )
+
+    def test_02_validation_fails_missing_job_id(self):
```

Comment:
I think the method name is left unchanged?

Reply:
Fixed